erpnext/projects/doctype/timesheet_detail/timesheet_detail.json

```diff
@@ -159,6 +159,7 @@
   },
   {
    "allow_on_submit": 1,
+   "default": "0",
```

Comment:
- Check the code which does the calculations for billable_hours
- This is a hack fix, need to fix the issue at the roow.

Reply:
@hrwX
* the code which calculate **billable hours** / **billing amount** works fine if the ***billing hours*** is avaliable.
* But while adding a new timesheet detail row there is no billing hours as of default in the json, due to which **billable hours** shows **NaN**, so to avoid that I gave 0 as default in place of **NaN**, else calculation is working good!